TTYC.IdentityServer/CustomProfileService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using IdentityServer4.Models;
+using IdentityServer4.Services;
+
+namespace TTYC.IdentityServer
+{
+    public class CustomProfileService : IProfileService
+    {
+        public Task GetProfileDataAsync(ProfileDataRequestContext context)
+        {
+            context.IssuedClaims = context.Subject.Claims.ToList();
+            return Task.CompletedTask;
+        }
+
+        public Task IsActiveAsync(IsActiveContext context)
```

Comment:
Looks like a place where you actually validate the LockoutEnd.
@lALTAIRl Please confirm